src/main/java/com/taskadapter/redmineapi/MembershipManager.java

```diff
@@ -46,6 +46,11 @@
         return transport.getChildEntries(Project.class, projectId, Membership.class);
     }
 
+    public List<Membership> getMemberships(int projectId, int countOfMembers) throws RedmineException {
+        transport.setObjectsPerPage(countOfMembers);
```

Comment:
this would permanently set number of objects in the transport object. this side-effect is not desirable for getMemberships() method.
we need another solution for this.

Reply:
If I will just add setObjectsPerPage(DEFAULT_VALUE) after getting memers list, would it be OK?

Reply:
setting this value on a shared "transport" affects all future usages of that class.
getMembership() or other functions should not have unexpected side-effects like this.

Reply:
OK, then I'll just make another function in transport with members count parameter.